app/main.py

```diff
@@ -1,6 +1,16 @@
 from typing import Callable
+import functools
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    results = {}
+
+    @functools.wraps(func)
+    def wrapper(*args, **kwargs) -> Callable:
+        if args not in results:
```

Comment:
you should check not only `args` but also `kwargs` together

Reply:
if I do `if args not in results and kwargs not in result` I have an error.

Reply:
Done.